Source/FeatureRegistry.swift

```diff
@@ -163,3 +163,26 @@ extension LabeledGroupItem { /* Collection Support */
     // This is a class (vs protocol) because we need to declare
     // internal conformance to FeatureExtractable for Obj-C compat.
 }
+
+extension FeatureRegistry {
+
+
+    /// Returns the label values for the enabled features in the provided registry.
+    /// Primarily used for debugging.
+    /// - Parameter featureRegistry: The feature registry that should be used to find enabled features
+    @objc public static func enabledFeatures(in featureRegistry: FeatureRegistry) -> [String] {
+
+        var enabledFeatures:[String] = []
+        featureRegistry.features.forEach { labeledItem in
+            if let labeledFeature = labeledItem as? LabeledFeatureItem, labeledFeature.feature.enabled {
+                enabledFeatures.append(labeledFeature.label)
+            } else if let labeledGroup = labeledItem as? LabeledGroupItem {
+                labeledGroup.forEach { item in
+                    guard let labeledFeature = item as? LabeledFeatureItem, labeledFeature.feature.enabled else { return }
```

Comment:
Actually FeatureGroup permits nested FeatureGroups, so this needs to work on arbitrary depth feature group trees, not only one level.

Reply:
Nested FeatureGroups appear in the UI but it doesn't look like they are configured correctly.

https://github.com/yahoo/Override/blob/5d8923cda38878faecfbb47bbdb06bf5f652ca29/Source/FeatureRegistry.swift#L126-L131

The code simply bails if the type isn't `LabeledFeatureItem` and nothing is done with a nested `FeatureGroup` object.

I'll create a separate PR to address that.

Reply:
Oh Crap! Good catch.